config/routes.rb

```diff
@@ -1,4 +1,9 @@
 Rails.application.routes.draw do
+  resources :users
```

Comment:
Restrict routes to what you have available:
resources users, only: [:new, :create, :whatever]